uctMaths/competition/interface/individual_as_template.html

```diff
@@ -42,140 +42,140 @@ <h1 style="text-align:center; font-size:18pt; font-family:'Univers',sans-serif;
 <body>
 
 
-    <table align = "center" width = 85% height = 7%>
-        <tr style="line-height: 125%; text-align:center; font-size:11pt; font-family:'Univers',sans-serif; padding: 3px; vertical-align: middle;">
-          <td width=8% ><b>CODE</b><br>{{code}}</td>
-          <td width=28% ><b>NAME</b><br>{{name}}</td>
-          <td width=28% ><b>SCHOOL</b><br>{{school}}</td>
-          <td width=8% ><b>GRADE</b><br>{{grade}}</td>
-          <td width=28% ><b>VENUE</b><br>{{venue}}</td>
+    <table class = "answersheet" align = "center" width = 85% height = 7%>
+        <tr class = "answersheet" style="line-height: 125%; text-align:center; font-size:11pt; font-family:'Univers',sans-serif; padding: 3px; vertical-align: middle;">
```

Comment:
I don't think there should be spaces between the `=` here so can you change `class = "answersheet"` to `class="answersheet"`